### ISSUE TITLE
acme.sh should NOT default to zerossl

### ISSUE THREAD

**Issue by Ressy66** [acmesh-official/acme.sh#3686]
Acme.sh should revert back to lets encrypt, as all LE certs are free.

no idea why this change was made, but really is a bad one - unless you now work for zerossl.

The new default zerossl, allows only THREE  90 day certs on the free plan, to get more or yearly cert, you pay 10 dollars US a month.
 
crazy,....

**Comment by Stellar-todd** [comment]
Agreed this move is pretty bad. Running into issues with our web server renewing old certs now

**Comment by hebbet** [comment]
that change was announced in early 2021 here.

https://community.letsencrypt.org/t/the-acme-sh-will-change-default-ca-to-zerossl-on-august-1st-2021/144052
https://github.com/acmesh-official/acme.sh/wiki/Change-default-CA-to-ZeroSSL

None of my old certs were moved zerossl when the cert got renewed. 

Running `acme.sh --set-default-ca  --server  letsencrypt` will make LE default for new certs too.

**Comment by hebbet** [comment]
> The new default zerossl, allows only THREE 90 day certs on the free plan

you get unlimited acme certs via api.

![image](https://user-images.githubusercontent.com/1423115/133033935-4bf456cc-9428-47eb-96fe-e474b73e630e.png)

**Comment by Ressy66** [comment]
we are fully aware it was announced, and how to change it, but teh decision too change it was not thought out very well. 
it was not consulted with the userbase - the users who make this project popular

the fact the change to  default  heavily limited CA over one that is completely unrestricted was very strange decision by Neil, very strange indeed.

**Comment by hebbet** [comment]
you know that apilayer bought both zerossl and acme.sh? so i my opinion comprehensible decision.
https://community.letsencrypt.org/t/the-acme-sh-will-change-default-ca-to-zerossl-on-august-1st-2021/144052/7

Neil didn't follow suggestions from the community in the past, etc. #1308

**Comment by Ressy66** [comment]
Neil has in the past been receptive, not always like DoH garbage, but yeah, as for ownership, no not aware as I do not read LE forums, not had a need to (I also prefer mailing lists - dont have to sift though all that useless html candy crap) but I suspected as much and kind of eluded to in my OP. 

Ahh well, might have to look elsewhere in future, if its becoming cripleware, it can only get worse

**Comment by Neilpang** [comment]
@Ressy66 

>> Neil has in the past been receptive, 

Yes,  and I will always be.

I have refused some of the suggestions/requests/ideas in the past,  but the accepted/fixed ones are far more than those that were refused.
I'm always listening to all the voices from all of you guys.  It's your voices that made acme.sh what it is now.  I appreciated any of such voices, no matter good or bad.

I'm always there to make acme.sh better, for you, for me and for anyone who liked, is liking, and will like acme.sh.

**Comment by tomiles** [comment]
Had to switch back to letsencrypt today, renewal with zerossl didn't work, either get 504 Gateway timeout or the renewal seems to work but "renewed" certificate is just the same expired one again.

**Comment by Arnie97** [comment]
@tomiles I met the same problem with zerossl today, and had to add `--force --server letsencrypt` to get a new certificate.

I'm glad to see the project got a commercial company as its sponsor, but they might still have some way to go.

**Comment by Neilpang** [comment]
Sorry for the troubles, guys. 
but let's give them some more time to be strong.
At the same time,  I'm here to fix any possible problems.  You can also switch back to use letsencrypt easily.

https://github.com/acmesh-official/acme.sh/wiki/Server

**Comment by lhorace** [comment]
@hebbet 

> you get unlimited acme certs via api.

That image doesn't read unlimited too me? Or am I missing something?

**Comment by hebbet** [comment]
> @hebbet
> 
> > you get unlimited acme certs via api.
> 
> That image doesn't read unlimited too me? Or am I missing something?

Why? It shows an Infinity Symbol

https://en.m.wikipedia.org/wiki/Infinity_symbol

**Comment by lhorace** [comment]
I didn't see the infinity symbol, would have preferred it spelled out. IE, terms of service etc. I'll have look at it later.

**Comment by hebbet** [comment]
Most TOS can change anytime without a notice. 

Docs says:
"By using ZeroSSL's ACME feature, you will be able to generate an unlimited amount of 90-day SSL certificates at no charge, also supporting multi-domain certificates and wildcards. Each certificate you create will be stored in your ZeroSSL account."

https://zerossl.com/documentation/acme/
